analyzers/src/SonarAnalyzer.Common/Rules/Hotspots/ExecutingSqlQueriesBase.cs

```diff
@@ -41,13 +43,63 @@ public abstract class ExecutingSqlQueriesBase<TSyntaxKind, TExpressionSyntax> :
+                KnownType.System_Data_Sqlite_SQLiteDataAdapter,
+            };
+
+        private readonly KnownType[] constructorsWithSecondParam =
+            {
+                KnownType.MySql_Data_MySqlClient_MySqlScript,
+            };
+
+        private readonly MemberDescriptor[] invocationsWithFirstTwoParams =
+            {
+                new MemberDescriptor(KnownType.Microsoft_EntityFrameworkCore_RelationalDatabaseFacadeExtensions, "ExecuteSqlCommandAsync"),
+                new MemberDescriptor(KnownType.Microsoft_EntityFrameworkCore_RelationalDatabaseFacadeExtensions, "ExecuteSqlCommand"),
+                new MemberDescriptor(KnownType.Microsoft_EntityFrameworkCore_RelationalQueryableExtensions, "FromSql"),
+            };
+
+        private readonly MemberDescriptor[] invocationsWithFirstParam =
+            {
+                new MemberDescriptor(KnownType.System_Data_Sqlite_SqliteCommand, "Execute"),
+            };
+
+        private readonly MemberDescriptor[] invocationsWithSecondParam =
+            {
+                new MemberDescriptor(KnownType.MySql_Data_MySqlClient_MySqlHelper, "ExecuteDataRow"),
+                new MemberDescriptor(KnownType.MySql_Data_MySqlClient_MySqlHelper, "ExecuteDataRowAsync"),
+                new MemberDescriptor(KnownType.MySql_Data_MySqlClient_MySqlHelper, "ExecuteDataset"),
+                new MemberDescriptor(KnownType.MySql_Data_MySqlClient_MySqlHelper, "ExecuteDatasetAsync"),
+                new MemberDescriptor(KnownType.MySql_Data_MySqlClient_MySqlHelper, "ExecuteNonQuery"),
+                new MemberDescriptor(KnownType.MySql_Data_MySqlClient_MySqlHelper, "ExecuteNonQueryAsync"),
+                new MemberDescriptor(KnownType.MySql_Data_MySqlClient_MySqlHelper, "ExecuteReader"),
+                new MemberDescriptor(KnownType.MySql_Data_MySqlClient_MySqlHelper, "ExecuteReaderAsync"),
+                new MemberDescriptor(KnownType.MySql_Data_MySqlClient_MySqlHelper, "ExecuteScalar"),
+                new MemberDescriptor(KnownType.MySql_Data_MySqlClient_MySqlHelper, "ExecuteScalarAsync"),
+                new MemberDescriptor(KnownType.MySql_Data_MySqlClient_MySqlHelper, "UpdateDataSet"),
+                new MemberDescriptor(KnownType.MySql_Data_MySqlClient_MySqlHelper, "UpdateDataSetAsync"),
+            };
+
+        private readonly MemberDescriptor[] properties =
+            {
+                new MemberDescriptor(KnownType.System_Data_Odbc_OdbcCommand, "CommandText"),
+                new MemberDescriptor(KnownType.System_Data_OracleClient_OracleCommand, "CommandText"),
+                new MemberDescriptor(KnownType.System_Data_SqlClient_SqlCommand, "CommandText"),
+                new MemberDescriptor(KnownType.System_Data_SqlServerCe_SqlCeCommand, "CommandText"),
+                new MemberDescriptor(KnownType.MySql_Data_MySqlClient_MySqlCommand, "CommandText"),
+                new MemberDescriptor(KnownType.Microsoft_Data_Sqlite_SqliteCommand, "CommandText"),
```

Comment:
I think there's and issue with `System.Data.Sqllite` and that is, that there exists
`System.Data.Sqlite` for .NET Fx and .NET Core 2.1
`System.Data.SQLite` for .NET Core 3.1 and .NET 5
It's documented in sonar-security ITs and I'm not sure if current configuration supports both (probably not yet).

Can you take a look and if it's confirmed, then add tests for netcore with `SQLite`?

Reply:
I could not find the nuget package version with System.Data.Sqlite.

Reply:
That was probably related to the `Microsoft.Data.Sqlite` => `System.Data.SQLite` move